### DISCUSSION TITLE
혹시 강의 녹강은 따로 올라오나요?

### DISCUSSION THREAD

**Discussion by cz5325** [dinnercoding/wanted-pre-onboarding-challenge-data#69] [Q&A] [answered]
11월18일 자격증 시험 일정 때문에 실강 참여가 힘들 거 같은데 혹시 따로 녹강을 들을 수 있는 방법이 있을까요?

**Comment by dinnercoding** [comment, marked as answer]
공지에 있는 것처럼 강의 녹화는 진행하지 않습니다. 
다만, 강의에 사용되는 자료들은 제공되오니 이 점 참고해주세요.
